build.gradle.kts

```diff
@@ -92,4 +92,9 @@ tasks {
       jvmTarget = "16"
     }
   }
+  compileTestKotlin {
+    kotlinOptions {
+      jvmTarget = "16"
+    }
+  }
```

Comment:
This would apply to both main and test code: https://github.com/ministryofjustice/hmpps-template-kotlin/blob/main/build.gradle.kts#L19